app/javascript/dashboard/components/widgets/conversation/conversationCardComponents/CardLabels.vue

```diff
@@ -2,10 +2,11 @@
   <div
     v-show="activeLabels.length"
     ref="labelContainer"
-    class="label-container mt-0.5 mx-2 mb-0"
+    v-resize="computeVisibleLabelPosition"
+    class="w-4/5 label-container"
   >
     <div
-      class="labels-wrap flex items-end min-w-0 flex-shrink gap-y-1 flex-wrap"
+      class="flex items-end flex-shrink min-w-0 labels-wrap gap-y-1"
       :class="{ expand: showAllLabels }"
```

Comment:
```suggestion
      class="flex items-end flex-shrink min-w-0 gap-y-1"
      :class="{ 'h-auto overflow-visible flex-row flex-wrap': showAllLabels }"
```

Let's try to avoid `@apply`